integrations/standalone/tests/pageobjects/Table.ts

```diff
@@ -84,7 +84,8 @@ export class Row {
   }
 
   async remove() {
-    await this.locator.getByRole('button', { name: 'Remove row' }).click();
+    await this.locator.click();
+    await this.page.getByRole('button', { name: 'Remove row' }).click();

```

Comment:
Yes, I have also considered that, but unfortunately, it doesn't work with the rowsLocator since this button is completely outside of the table. I wouldn't know how to make it more specific, as the button simply belongs to the collapsible-header div, and there can be multiple of them on a page.

Reply:
Leave it as it is for now 👍